Source/Core/VideoCommon/ShaderGenCommon.h

```diff
@@ -90,19 +82,30 @@ class ShaderUid : public ShaderGeneratorInterface
     return memcmp(this->values, obj.values, data.NumValues() * sizeof(*values)) < 0;
   }
 
-  template <class uid_data2>
-  uid_data2* GetUidData()
-  {
-    return &data;
-  }
+  /**
```

Comment:
This file seems to mix `// comments` with `/* comments */`...thoughts on making those few down here `// comments` for terseness?

Reply:
Sure thing. Just amended it.